tests/src/test/scala/com/nvidia/spark/rapids/RegularExpressionTranspilerSuite.scala

```diff
@@ -184,6 +176,12 @@ class RegularExpressionTranspilerSuite extends FunSuite with Arm {
         "\u0007\u003f\u007f", "\u007f", "\u007f2"))
   }
 
+  test("octal digits >= 0o177 - find") {
```

Comment:
I think should be merged this with other octal digit unit test to test all octal digits in one unit test.